addons/project/models/project.py

```diff
@@ -250,6 +260,12 @@ def _get_default_favorite_user_ids(self):
         ('quarterly', 'Quarterly'),
         ('yearly', 'Yearly')], 'Rating Frequency', required=True, default='monthly')
 
+    # stage fields
+    stages_active = fields.Boolean('Project Stages', default=lambda self: self.env.user.has_group('project.group_project_stages'))
```

Comment:
Shouldn't be a field, can be accessed directly via ``self.env.user.has_group('project.group_project_stages')``